remote-api/src/main/java/org/alfresco/rest/api/impl/rules/RuleSetsImpl.java

```diff
@@ -110,6 +111,22 @@ public RuleSetLink linkToRuleSet(String folderNodeId, String linkToNodeId)
         return ruleSetLink;
     }
 
+    @Override
+    public void unlinkRuleSet(String folderNodeId, String ruleSetId)
+    {
+        final NodeRef folderNodeRef = validator.validateFolderNode(folderNodeId,true);
+        final NodeRef ruleSetNodeRef = validator.validateRuleSetNode(ruleSetId, folderNodeRef);
+
+        //The folder should be linked to a rule set
+        if (!ruleService.isLinkedToRuleNode(folderNodeRef))
+        {
+            throw new InvalidArgumentException("The folder is not linked to a rule set.");
+        }
+
+        //The following line also handles the deletion of the child folder that gets created during linking

```

Comment:
Do you mean that it gets rid of the parent-child association?  Presumably it doesn't get rid of the rule folder.